kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -193,16 +204,18 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
       endTime: Long,
       from: Int,
       size: Int): Seq[Batch] = {
-    metadataManager.getBatches(batchType, batchUser, batchState, createTime, endTime, from, size)
+    metadataManager.map(
+      _.getBatches(batchType, batchUser, batchState, createTime, endTime, from, size))
+      .orNull
```

Comment:
is orNull a good choice, where NPE lurks

Reply:
I just follow the semantics of original code, the test will fail if change to Seq.empty.. I do not want to do more change in this pr so use `orNull`.

cc @turboFei

Reply:
which is the failed test

Reply:
Not sure which code cause test failed .. I wil try to fix it in this pr

Reply:
@yaooqinn  it's ok now